iothub/device/src/Transport/Mqtt/IWillMessage.cs

```diff
@@ -1,23 +1,21 @@
-// Copyright (c) Microsoft. All rights reserved.
+﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using DotNetty.Codecs.Mqtt.Packets;
-
 namespace Microsoft.Azure.Devices.Client.Transport.Mqtt
 {
     /// <summary>
-    /// An MQTT "will" message, used when a client disconnects
+    /// An MQTT "will" message to be sent by this client when the client disconnects.
     /// </summary>
     public interface IWillMessage
     {
         /// <summary>
-        /// The message to be sent
+        /// The payload to be sent in the will message
         /// </summary>
-        Message Message { get; }
```

Comment:
I'm happy that we're finally moving away from using IoT `Message` to represent an MQTT message.